web-app/src/pages/for-employers/for-employers.tsx

```diff
@@ -1,232 +1,388 @@
 import React from "react";
 import styled from "styled-components";
-
+import { Row, Col, Table, Card } from "react-bootstrap";
 import PageWrapper from "src/components/page-wrapper";
-import Banner, { BannerTitle } from "src/components/banner";
-import { LinkButton } from "src/components/button";
+import { LinkButton, WhiteButton } from "src/components/button";
 import { forSize } from "src/responsive";
 import * as urlGenerators from "src/pages/url-generators";
 import colors from "src/colors";
 import { translations } from "src/translations";
 import { useState, useEffect } from "react";
-import { WhiteButton } from "src/components/button";
+import ReactStars from "react-rating-stars-component";
+import ResourcePreviewCard from "./ResourcePreviewCard";
+import ResourceTopicButton from "./ResourceTopicButton";
 import resourcesIcon from "src/images/icons/resources-icon.png";
+import bannerImage from "../../images/employer-banner-right-section.png";
+import arrowDownImage from "../../images/arrow-down-circle-line.png";
+import facebookImage from "../../images/facebook.png";
+import getResultImage from "../../images/getResult.png";
+import saveMoneyImage from "../../images/saveMoney.png";
+import saveTimeImage from "../../images/saveTime.png";
+import dollarImage from "../../images/job-post-icons/dollar.png";
+import addressImage from "../../images/job-post-icons/address.png";
+import languageImage from "../../images/job-post-icons/language.png";
+import certificateImage from "../../images/job-post-icons/certificate.png";
+import cityImage from "../../images/job-post-icons/city.png";
+import descriptionImage from "../../images/job-post-icons/description.png";
+import industrialParkImage from "../../images/job-post-icons/industrial-park.png";
+import industryImage from "../../images/job-post-icons/industry.png";
+import jobTypeImage from "../../images/job-post-icons/job-type.png";
+import minEducationImage from "../../images/job-post-icons/min-education.png";
+import shiftsImage from "../../images/job-post-icons/shifts.png";
+import skillsImages from "../../images/job-post-icons/skills.png";
+import arrowDownCircleImage from "../../images/arrow-down-circle-line.png";
+import navigationArrowImage from "../../images/job-post-icons/navifationarrow.png";
+import topic1Image from "../../images/job-post-icons/topic-1.png";
+import topic2Image from "../../images/job-post-icons/topic-2.png";
+import topic3Image from "../../images/job-post-icons/topic-3.png";
+import img1 from "../../images/workers.jpeg";
```

Comment:
It doesn't look pretty to have all of these images be imported in separate lines. Would it be possible to import all of the images needed from one location? You might need to create a file that exports all of the images in one files so that you can import all of the images from that single file.